scripts/ConvertToAckermann.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+"""
+Subscribes to a twist message and converts it to an ackermann message.
+Subscribes to /twist_vel
+Publishes to /cmd_vel
+
+Assumes all inputs are normalized between -1 and 1
+
+@edited: 9/28/18
+@author: Amy Phung
+"""
+
+import rospy
+from geometry_msgs.msg import Twist
+from ackermann_msgs.msg import AckermannDrive
+from scipy.interpolate import interp1d
+
+class ConvertToAckermann():
+    def __init__(self):
+        # Define variables
+        self.twist_data = None
+
+        # Define ROS constructs
+        rospy.init_node("convert_to_ackermann")
+        self.twist_sub = rospy.Subscriber("/twist_vel", Twist, self.twist_cb)
+        self.ack_pub = rospy.Publisher("/drive", AckermannDrive, queue_size=10)
```

Comment:
If we are doing this for control, `queue_size=1` so that we are always using the last command.